### PR TITLE
Add reset zoom to overall survival chart

### PR DESCRIPTION
**Pull request by jaouad-benassila**

### CODE REVIEW

File: src/components/CohortBuilder/Summary/SurvivalChart.js [hunk 1]
```diff
@@ -120,7 +131,10 @@ class SurvivalPlot extends React.Component {
           onMouseLeaveDonors:
             this.props.onMouseLeaveDonors && this.props.onMouseLeaveDonors.bind(this),
           onClickDonors: this.props.onClickDonors,
-          onDomainChange: newXDomain => this.updateState({ xDomain: newXDomain }),
+          onDomainChange: newXDomain => {
+            this.props.disableResetButton() && this.props.disableResetButton.bind(this);
```
Comment (jberube) [inline src/components/CohortBuilder/Summary/SurvivalChart.js]: Can you explain what this line does?

Comment (jaouad-benassila) [reply src/components/CohortBuilder/Summary/SurvivalChart.js]: to disable  reset zoom button , I think is clear

Comment (jberube) [reply src/components/CohortBuilder/Summary/SurvivalChart.js]: Yeah, what it does its clear.

What preocupies me is the second part of the line: 
` && this.props.disableResetButton.bind(this);`

From what I understand, `this.props.disableResetButton()` will be `handleDisableResetButton` at line #298 which never returns any value:
```
  handleDisableResetButton = () => {
    this.setState({ zoomDisabled: false });
  };
```

So why the ` this.props.disableResetButton() && this.props.disableResetButton.bind(this);` and not just `this.props.disableResetButton()`?

**Comment by aalex** [comment]
I had to use my admin priviledges to merge this PR. @nyanofthemoon recommended to merge it. Please create a refactoring backlog item if needed. (to address the changes needed according to the conversation around this PR)

**Pull request merged**